### PR TITLE
Update UserGuide

### LINKED ISSUE #220: [PE-D] Make the guide for feature "items update" complete
The example input show on the UG for this feature is not exactly the same with the actual input. The step of selecting the item to update by index is omitted. It would be good to include everything as it is in the UG.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: geezzzyyy/ped#2

### LINKED ISSUE #238: [PE-D] Unclear documentation on items update
Initially I thought u were suppose to type items update p in one line and realised is not  that.

Instead perhaps it better to show in full how the code is suppose to work i.e the whole chunk on what you are suppose to type

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: Chihui8199/ped#5

### PR DESCRIPTION
**Pull request by Cocokkkk**
Add descriptions of help, items, orders, items add, items list and items update.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ Section 2 will explain the features available in easyLog.
 
 ### 2.1 Viewing help: `help`
 
-Shows available commands of the app.
+As a first-time user, you can simply type `help` to view all available commands of the app.
```
Comment (ongweisheng) [inline docs/UserGuide.md]: Maybe consider adding also if you forget some of the available commands or need to check their format they can use the help function as well?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -139,7 +139,8 @@ Section 2.2 will explain the features related to items.
 
 #### 2.2.1. Show items-related commands: `items`
 
-Show commands related to items and brief description of the commands: add item, list items, remove items, clear items
+As a first-time user, you can simply type `items` to view all commands related to items. At the same time, you can see
+the brief description of those commands.
```
Comment (ongweisheng) [inline docs/UserGuide.md:121]: same as above, but i feel this one is more relevant towards just looking at items related features if the user forgets

File: docs/UserGuide.md [hunk 3]
```diff
@@ -162,11 +163,15 @@ ____________________________________________________________
...
+Are you sometimes overwhelmed by handling a large variety of items? No worries, (items add) feature can help you.
+Let's just assume you are a home-based baker and you have a variety of cookies to sell. Now, you are going to record
+the first type of cookie - brownie. The price of each brownie is 5 dollars and the stock of brownie is 50. You just 
+need to type `items add` followed by item name `brownie`, item price `5` and item stock `50`, then press enter. Done! 
+You can see the confirmation message when the item is recorded successfully. Just like this, you can record more items.
 
 Format: `items add <item_name>` followed by `<item_price>` and `<item_stock>`
 
-Example: `items add PS5` followed by `100` and `10`
+Example: `items add brownie 5 50`
```
Comment (ongweisheng) [inline docs/UserGuide.md]: think this is wrong because we can't add as 1 line

File: docs/UserGuide.md [hunk 4]
```diff
@@ -162,11 +163,15 @@ ____________________________________________________________
 
 #### 2.2.2. Add an item: `items add`
 
-Add an item to the system.
+Are you sometimes overwhelmed by handling a large variety of items? No worries, (items add) feature can help you.
```
Comment (ongweisheng) [inline docs/UserGuide.md]: maybe do step by step instead of talking in story form, not too sure, we can discuss again

File: docs/UserGuide.md [hunk 5]
```diff
@@ -330,8 +327,8 @@ Section 2.3 will explain the features related to orders.
 
 #### 2.3.1. Show orders-related commands : `orders`
 
-Show commands related to orders and brief description of the commands:
-  add order, list orders, remove orders, clear orders
+As a first-time user, you can simply type `orders` to view all commands related to orders. At the same time, you can see
```
Comment (ongweisheng) [inline docs/UserGuide.md]: same as above comments regarding help and items

**Review by ongweisheng — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md [hunk 6]
```diff
@@ -97,114 +97,67 @@ Section 2 will explain the features available in easyLog.
...
+1. Input `items` command.
+2. Press Enter.
+3. All available commands related to items will then be shown.
 
 #### 2.2.2. Add an item: `items add`
 
-Add an item to the system.
-
-Format: `items add <item_name>` followed by `<item_price>` and `<item_stock>`
+When you sometimes overwhelmed by handling a large variety of items, `items add` feature allows you to record the
```
Comment (ongweisheng) [inline docs/UserGuide.md]: hi can fix grammar error here? When you are*?

File: docs/UserGuide.md [hunk 7]
```diff
@@ -97,114 +97,67 @@ Section 2 will explain the features available in easyLog.
...
+2. You will be prompted to key in the item price and the item stock.
+3. Input in this format `<item_price> <space> <item_stock>` for the price and stock to be added to the items details.
+4. Press enter.
+5. You can see the confirmation message when the item is recorded successfully.
 
-💡 item names cannot be empty or consist of only white spaces. <br>
-💡 item price must be a number ranging from 0 to 1,000,000,000 (inclusive). <br>
-💡 item stock has to be a integer ranging from 0 to 1,000,000,000 (inclusive). <br>
+Note:
+> 📄 item names cannot be empty or consist of only white spaces. <br>
```
Comment (ongweisheng) [inline docs/UserGuide.md:142]: consider adding a limit to the item names as I received a feedback saying the orders name is unlimited and causes a bug when storing the long name into save file. Limit that i set for customer name for orders add was 30 characters including spaces

File: docs/UserGuide.md [hunk 8]
```diff
@@ -97,114 +97,67 @@ Section 2 will explain the features available in easyLog.
...
+> 📄 item price must be a number ranging from 0 to 1,000,000,000 (inclusive). <br>
+> 📄 item stock has to be an integer ranging from 0 to 1,000,000,000 (inclusive). <br>
+> 📄 When in doubt, do follow the prompts shown on the command terminal. <br>
 
 #### 2.2.3. Show the item list: `items list`
 
-You can use this command to view all the recorded items in the system.
+After your business has been running successfully for a period of time, whenever you would like to check all items in
```
Comment (ongweisheng) [inline docs/UserGuide.md]: Maybe just say whenever you would like to check the items and their stock in your inventory is enough already?

**Review by ongweisheng — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 7 above)
Comment (Cocokkkk) [reply docs/UserGuide.md:142]: Okay! Thank you for your review!

**Review by ongweisheng — APPROVED** [review]

**Pull request merged**